src/AMY-Arduino.h

```diff
@@ -3,9 +3,9 @@
 
 #ifndef AMYARDUINOH
 #define AMYARDUINOH
+//#include <Adafruit_TinyUSB.h>
 
-
-#include "Arduino.h"
+//#include "Arduino.h"
```

Comment:
This makes me nervous.  We may not need it, but we should be able to include it.